sgt-back-end/index.js

```diff
@@ -0,0 +1,111 @@
+const pg = require('pg');
+const express = require('express');
+const app = express();
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+// Declaring our Middleware
+const jsonMiddleware = express.json();
+app.use(jsonMiddleware);
+// Get our tables.
+app.get('/api/grades', (req, res) => {
+  const sql = ` SELECT *
+  FROM "grades" `;
+
+  db.query(sql)
+    .then(result => {
+      const array = [];
+      for (var i = 0; i < result.rows.length; i++) {
+        array.push(result.rows[i]);
+      }
+      const grade = result.rows;
+      if (!grade[0]) {
+        res.status(404).json({
+          error: 'Cannot find grade any rows'
+        });
+      } else {
+        res.json(array);
+      }
```

Comment:
You don't need a loop here you can just directly use `result.rows` or directly assign it into another variable with `const grades = result.rows`

```JAVASCRIPT
const grades = result.rows;

if (!grades.length) {
  res.status(404).json({
    error: 'Cannot find grade any rows'
  });
} else {
  res.json(grades);
}
```